kubernetes/etc/kubernetes/k8s.node.rc

```diff
@@ -4,6 +4,6 @@ stuff "until nc -z __MASTER_IP__ 8080; do sleep 10; done; /home/ubuntu/kubernete
 screen -t kube-proxy
 select 1
 stuff "until nc -z __MASTER_IP__ 8080; do sleep 10; done; /home/ubuntu/kubernetes/cluster/ubuntu/binaries/minion/kube-proxy --master=http://__MASTER_IP__:8080^M"
-screen -t network-policy
+screen -t network-policy-agent
 select 2
-stuff "until curl -s http://__MASTER_IP__:8080/apis/romana.io/demo/v1/namespaces/default/networkpolicys/ | grep -q policy; do sleep 10; done; python /home/ubuntu/romana/kubernetes/k8s-listener.py^M"
+stuff "until curl -s http://__MASTER_IP__:8080/apis/romana.io/demo/v1/namespaces/default/networkpolicys/ | grep -q policy; do sleep 10; done; python /home/ubuntu/romana/kubernetes/k8s-listener.py --agent^M"
```

Comment:
Question about this curl request: Is this just so that we know when the event URL was provisioned and is ready? Is it correct to do this in the installation scripts? It seems that this behaviour should go into the listener itself. That way, it could gracefully handle even occasional issues with the kube API, where maybe intermittently the API became unavailable. Basically: If there is an immediate reset or a timeout (no connection established), the listener could switch into a slow-polling mode...